### ISSUE TITLE
Undefined control sequence when used with beamer

### ISSUE THREAD

**Issue by nickw2066** [zepinglee/citeproc-lua#41]
**Describe the bug**
"! Undefined control sequence.
\l__csl_citation_tl ->\cslcite
{McCarthy2007-Deut.}{Carmel McCarthy, ed., \t...
l.13 \end{frame}"

**Additional information**
- TeX distribution: MacTeX 2023
- Package `citation-style-language` version: 0.4.2
- LaTeX engine: `lualatex`

**To Reproduce**
.tex file:
```tex
% !TEX encoding = UTF-8 Unicode
\documentclass[aspectratio=169]{beamer}

\usepackage{citation-style-language}
\cslsetup{style = turabian-fullnote-bibliography-no-ibid}
\addbibresource{myBib.json}

\begin{document}

\begin{frame}
\frametitle{Test MWE}
Hello\cite{McCarthy2007-Deut.}
\end{frame}

\end{document}
```

.json file:
```json
[
    {
        "id": "McCarthy2007-Deut.",
        "type": "book",
        "editor": [
            {
                "family": "McCarthy",
                "given": "Carmel"
            }
        ],
        "collection-editor": [
            {
                "family": "Schenker",
                "given": "A."
            }
        ],
        "title": "Deuteronomy",
        "title-short": "Deut.",
        "source": "Accordance",
        "collection-title": "Biblia Hebraica Quinta",
        "collection-number": "5",
        "publisher-place": "Stuttgart",
        "publisher": "Deutsche Bibelgesellshaft",
        "issued": {
            "date-parts": [
                [
                    "2007"
                ]
            ]
        }
    }
]

```
Here is the output:
[mwe.pdf](https://github.com/zepinglee/citeproc-lua/files/12490716/mwe.pdf)

**Comment by zepinglee** [comment]
Thanks for your feedback and I can reproduce the error. It's basically the compatibility problem with both `beamer` and `hyperref`.

**Comment by nickw2066** [comment]
Thank you for your answer. Is the error fixable?

**Comment by zepinglee** [comment]
Fixed in a05611d.

There are still conflicts when csl is used with `beamer` and `backref` since all them modify the internal bibliographic commands of LaTeX2e). I'll try to resolve this before releasing to CTAN.

**Comment by nickw2066** [comment]
Thank you, I updated to "citation-style-language 2023-09-07 v0.4.3" and the MWE above now works fine.
However, if I change the MWE to
```
% !TEX encoding = UTF-8 Unicode
\documentclass[aspectratio=169]{beamer}

\usepackage{citation-style-language}
\cslsetup{style = turabian-fullnote-bibliography-no-ibid}
\addbibresource{myBib.json}

\begin{document}

\begin{frame}
\frametitle{Test MWE}
Hello\cite{McCarthy2007-Deut.}
\end{frame}

\begin{frame}
\printbibliography
\end{frame}

\end{document}
```
(that is, if I add the frame with the \printbibliography command) I get the following error: "! Undefined control sequence.\beamer@bibitem #1->\@bibitem{#1}\ignorespaces \end{frame}".
The footnote is still fine, but the bibliography looks like: <see attached>
<img width="531" alt="Screenshot 2023-09-12 at 7 21 48 pm" src="https://github.com/zepinglee/citeproc-lua/assets/110715564/127ec173-b612-48d7-aab7-2b022932bb97">

**Comment by zepinglee** [comment]
It turns out to be a typo that has just been fixed in cc77314 . I'll upload to CTAN then.

**Comment by nickw2066** [comment]
Thank you!
